apps/transport/lib/transport_web/controllers/aoms.ex

```diff
@@ -61,7 +61,11 @@ defmodule TransportWeb.AOMSController do
   defp valid_dataset?(dataset), do: Enum.any?(dataset.resources, fn r -> !Resource.is_outdated?(r) end)
 
   defp up_to_date?([]), do: nil
-  defp up_to_date?(datasets), do: Enum.any?(datasets, &valid_dataset?/1)
+  defp up_to_date?(datasets) do
+    datasets
+    |> Enum.filter(fn d -> Resource.is_transit_file?(d.type) end)
+    |> Enum.any?(&valid_dataset?/1)
```

Comment:
hum if the array is empty there, `Enum.any?` will return false, it is not what you want no?

Reply:
Indeed! fixed